packages/framework/fluid-framework/src/index.ts

```diff
@@ -34,6 +34,27 @@ export type {
 	Myself,
 } from "@fluidframework/fluid-static";
 export type { SharedObjectKind } from "@fluidframework/shared-object-base";
+export type {
+	IErrorBase,
+	IEventProvider,
+	IDisposable,
+	IEvent,
+	IEventThisPlaceHolder,
+	IErrorEvent,
+	ErasedType,
+	IFluidHandle,
+	IFluidLoadable,
+	ITelemetryBaseProperties,
+	IEventTransformer,
+	IProvideFluidLoadable,
+	IFluidHandleErased,
+	TransformedEvent,
+	TelemetryBaseEventPropertyType,
+	Tagged,
+	ReplaceIEventThisPlaceHolder,
+	FluidObject, // Linked in doc comment
```

Comment:
What item links this? Just wondering if it might be better to just not have that link in the docs (especially if it's something like an interface referring to once of its implementations).

Reply:
The doc comment on IFluidHandle is mostly just a link to this type. Its not clean to me who might need to use FluidObject, but this is clearly an area that could use some improvement: I just don't know what the docs should say.